pyk/src/pyk/tests/test_pretty_print_kast.py

```diff
@@ -17,14 +26,16 @@ class PrettyPrintKastTest(TestCase):
         (KRule(Bool.true), 'rule  true\n  '),
         (KRule(Bool.true, ensures=Bool.true), 'rule  true\n  '),
         (KRule(Bool.true, ensures=KApply('_andBool_', [Bool.true, Bool.true])), 'rule  true\n   ensures ( true\n   andBool ( true\n           ))\n  '),
+        (KProduction(KSort('Test')), 'syntax Test'),
+        (KProduction(KSort('Test'), att=KAtt({'token': ''})), 'syntax Test [token()]'),
```

Comment:
Please add a test for when there is a right-hand side.

Reply:
Done. Good catch.